hack/build.sh

```diff
@@ -61,8 +61,8 @@ function build_caa_payload_image() {
 		--build-arg YQ_CHECKSUM="${YQ_CHECKSUM}" \
 		-f Dockerfile \
 		${tag_string} \
-		--push \
```

Comment:
here maybe we need `--load` to replace `--push`

Reply:
adding `--load` works for podman
I cannot identify a difference between running with or without it.

Reply:
docker buildx build docker image inside one container instance, eg.
```
docker buildx ls
NAME/NODE        DRIVER/ENDPOINT             STATUS  BUILDKIT             PLATFORMS
multi-builder    docker-container
  multi-builder0 unix:///var/run/docker.sock running v0.12.5              linux/amd64*, linux/s390x*, linux/amd64/v2, linux/amd64/v3, linux/amd64/v4, linux/386
default *        docker
  default        default                     running v0.11.6+0a15675913b7 linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/amd64/v4, linux/386
```
Without the `--load` option, the built out image only inside the container can't be list on the host machine, so the new `docker push  ${registry}/${name}` will be failed, as it try to push image from the host.

Reply:
Added --load